Dockerfile

```diff
@@ -22,10 +22,11 @@ ENV BUNDLE_APP_CONFIG=$BUNDLE_PATH \
 ENV PATH=/app/bin:$BUNDLE_BIN:$PATH
 
 ENV FRAMEWORKS $FRAMEWORKS
+ENV RUBY_IMAGE $RUBY_IMAGE
 
-RUN mkdir -p $VENDOR_PATH \
-      && chown -R $USER_ID_GROUP $VENDOR_PATH
-
+# Copy cached folder to speed up docker containers
+COPY vendor /vendor
+RUN chown -R $USER_ID_GROUP /vendor
```

Comment:
This is required to use the cached artifacts if required, otherwise the docker build uses the `VENDOR_PATH` but the docker run overrides it, as a consequence it fails in the CI.

When building a docker image there is now way to mount volumes but using the `COPY`, this will help to reuse the existing local cache artifacts.